tests/test_other.py

```diff
@@ -7852,3 +7852,39 @@ def test_emar_M(self):
     result = subprocess.check_output([PYTHON, EMAR, 't', 'combined.a'])
     assert 'file1' in result
     assert 'file2' in result
+
+  def test_emar_append_hashes(self):
```

Comment:
which of these tests would fail before the changes in this PR?

Reply:
`test_emar_append_hashes` would pass before these changes, as it tests already existing hashing and ensures I didn't break it.

both `test_emar_append_hashes_rsp` and `test_emar_append_hashes_rsp_recursive` would fail before the changes.

`test_emar_append_hashes_rsp` tests hashing when response files are used, and `test_emar_append_hashes_rsp_recursive` tests recursive traverse of nested response files.